hugegraph-core/src/main/java/com/baidu/hugegraph/backend/serializer/SerializerFactory.java

```diff
@@ -32,6 +34,35 @@
         serializers = new ConcurrentHashMap<>();
     }
 
+    public static AbstractSerializer serializer(String name, HugeConfig config) {
```

Comment:
it's ok to just add `HugeConfig config` to `serializer(String name)` as the first param